### ISSUE TITLE
Container apps missing parameters

### ISSUE THREAD

**Issue by Alfonsoaz01** [hashicorp/terraform-provider-azurerm#22755]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Community Note

* Please vote on this issue by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment and review the [contribution guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/README.md) to help.

### Terraform Version

1.0.0

### AzureRM Provider Version

3.67

### Affected Resource(s)/Data Source(s)

azurerm_container-_apps

### Terraform Configuration Files

```hcl
provider
```

### Debug Output/Panic Output

```shell
│ Error: expected ingress.0.transport to be one of [auto http http2], got Tcp
│
│   with azurerm_container_app.container_app_redis,
│   on Application.tf line 58, in resource "azurerm_container_app" "container_app_redis":
│   58:     transport = "Tcp"
```

### Expected Behaviour

It should deploy container app

### Actual Behaviour

It does not deploy and it says it cannot be used

### Steps to Reproduce

1. apply

**Comment by Alfonsoaz01** [comment]
It should accept tcp as a parameter but it does not: https://learn.microsoft.com/en-us/azure/templates/microsoft.app/containerapps?pivots=deployment-language-terraform

**Comment by rcskosir** [comment]
These may be related: #21133 #22804

**Comment by SpartakusMd** [comment]
Fixed in #23752. Could you confirm it's working?

**Comment by Alfonsoaz01** [comment]
Hi @SpartakusMd ! I have checked that it works now. Thank you so much!
